discovery-infra/utils.py

```diff
@@ -255,6 +282,14 @@ def wait_till_cluster_is_in_status(
         log.info("Cluster: %s", client.cluster_get(cluster_id))
         raise
 
+def is_cluster_in_status(client, cluster_id, statuses):
```

Comment:
move line 276 to work with this function

Reply:
changed